content/03-types/03-document/01-what-are-document-dbs.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'What are document databases?'
+metaTitle: "What are document databases? | Prisma's Data Guide"
+metaDescription: 'In this article, we examine what document databases are and their advantages and tradeoffs'
+metaImage: '/social/docs-social.png'
+authors: ['alexemerich']
+---
+
+## Introduction
+The storage and organization of data are critical to the success of applications. Methods have evolved since the days of punch cards and player pianos. [Relational databases](/intro/database-glossary#relational-database) keeping data stored in a series of [rows](/intro/database-glossary#row) and [columns](/intro/database-glossary#column) among joined [tables](/intro/database-glossary#table) has been the overwhelming preferred choice for the last decades. These databases rely on [structured query language (SQL)](/intro/database-glossary#sql) to access the information and communicate its results to the requester.
+
+As application design has continued developing, new databases have become increasingly popular for their different strengths. In this guide, we will cover one of the popular [NoSQL](/intro/database-glossary#nosql) database types, document-oriented databases. We will discuss what they are and where they came from, how [documents](/intro/database-glossary#document) work, their features, and their advantages and disadvantages. 
+
+## What are document databases?
+[Document databases](/intro/database-glossary#document-database) are a NoSQL database that stores data as JSON documents instead of columns and rows like in a SQL relational database. They are a subclass of the key-value store NoSQL database concept. Document databases deliver a better developer experience because of their closeness to modern programming techniques. It is easy to read JSON, and it is translatable to the languages developers are most often writing today.
```

Comment:
Also:

* You mention XML documents later on, but here you only mention JSON.  Should we do a "JSON and other data serialization formats" type of thing or is there another way to either keep it broad here or make it clear that JSON is one implementation?
* I'd try to avoid definitive statements like "deliver a better developer experience" if possible to side step value judgments.  Maybe adding something like "seek to" would help with that.